### PR TITLE
[TF FE] Review of operations: batch 5

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
Ops in the batch:
- Const
- FakeQuantMinMaxVars
- FusedBatchNorm
- Interpolate
- L2Loss
- LRN
- NonMaxSuppression
- OneHot

### Tickets:
 - *70339*

### CODE REVIEW

File: ngraph/frontend/tensorflow/src/op/interpolate.cpp [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/interpolate.cpp:7]: line 39 Does input always 4D? Please ass assert if it is True
line 30 Remove TODO
line 32 Does using input.get_shape break reshapablity? Could we create ShapeOf subgraph instead?

File: ngraph/frontend/tensorflow/src/op/lrn.cpp [hunk 2]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/lrn.cpp:7]: line 23 Not clear comment. Do we need to do changes due to difference between Tf and OV or why it is here?
line 30 Implement and remove TODO (or convert to tickets)

File: ngraph/frontend/tensorflow/src/op/fake_quant_min_max_vars.cpp [hunk 3]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
```
Comment (evolosen) [inline ngraph/frontend/tensorflow/src/op/fake_quant_min_max_vars.cpp:6]: need  support dynamic case

File: ngraph/frontend/tensorflow/src/op/fused_batch_norm.cpp [hunk 4]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
```
Comment (evolosen) [inline ngraph/frontend/tensorflow/src/op/fused_batch_norm.cpp:5]: Need support dynamic case

File: ngraph/frontend/tensorflow/src/op/interpolate.cpp [hunk 5]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
```
Comment (evolosen) [inline ngraph/frontend/tensorflow/src/op/interpolate.cpp:5]: Need support dynamic case

File: ngraph/frontend/tensorflow/src/op/non_max_suppression.cpp [hunk 6]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/non_max_suppression.cpp:5]: line 32,46 the same about TODO

**Pull request closed without merging**